glean-core/preview/src/configuration.rs

```diff
@@ -7,8 +7,6 @@
 /// Optional values will be filled in with default values.
 #[derive(Debug, Clone)]
 pub struct Configuration {
-    /// Whether upload should be enabled.
-    pub upload_enabled: bool,
```

Comment:
Are you sure this is actually required, in this case? This is basically an external package/API, we're changing internals in other places.